language/stdlib/transaction_scripts/create_designated_dealer.move

```diff
@@ -19,9 +21,20 @@ fun create_designated_dealer<Currency>(
         tc_account, addr, auth_key_prefix, add_all_currencies
     );
     // Create default tiers for newly created DD
-    DesignatedDealer::add_tier(tc_account, addr, 500000);
-    DesignatedDealer::add_tier(tc_account, addr, 5000000);
-    DesignatedDealer::add_tier(tc_account, addr, 50000000);
-    DesignatedDealer::add_tier(tc_account, addr, 500000000);
+    if (add_all_currencies) {
```

Comment:
Could/should we move the tier addition inside `DesignatedDealer`? I  think that would make it easier to establish module invariants about the existence of tiers. This would also let us abstract `add_tier` into a function and save some code duplication.

Reply:
I agree, also since we have to add specific bounds when adding tiers, it makes sense to keep these constants (which will probably change) in `DesignatedDealer`

Reply:
Sounds good! I'll move this to the DD module

Reply:
@tzakian : I have a PR in progress that makes changes to DesignatedDealer.move.  If you try to touch it too, we will have a hard time landing.  I can make this change myself.  Is it ok for you to land this PR without making changes to Designated Dealer?